ArtOfIllusion/src/artofillusion/material/LinearMaterialMapping.java

```diff
@@ -45,7 +45,8 @@ public String getName()
     return "Linear";
   }
 
-  public static boolean legalMapping(Object3D obj, Material mat)
+  @Override
+  public boolean legalMapping(Object3D obj, Material mat)
```

Comment:
But is there any benefit to it other than simplifying a very small amount of code (a few dozen lines total)?  That doesn't seem like a good reason to change a public API in a way that breaks compatibility.